save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/MyProjectsRating.kt

```diff
@@ -0,0 +1,83 @@
+package com.saveourtool.save.frontend.components.views.contests
+
+import com.saveourtool.save.entities.ContestDto
+import com.saveourtool.save.entities.Project
+import com.saveourtool.save.frontend.externals.fontawesome.faUser
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.info.UserInfo
+import csstype.*
+import kotlinx.js.jso
+import org.w3c.fetch.Response
+import react.*
+import react.dom.html.ReactHTML
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.h4
+import react.dom.html.ReactHTML.p
+
+/**
+ * [Props] of the project rating functional interface
+ */
+external interface ProjectRatingProps : PropsWithChildren {
+    /**
+     * logged-in user or null
+     */
+    var userInfo: UserInfo?
+}
+
+// TODO: investigate why user properties are not propagated into the FC if we refresh the page (F5),
```

Comment:
@Cheshiriks @sanyavertolet @petertrr I am pretty sure that you have faced this kind of issue already - pls suggest something, if you know it

Reply:
resolved with useDeferredRequest